src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplatePhrase/PhrasalTemplatePhrase.tsx

```diff
@@ -105,6 +117,16 @@ export const PhrasalTemplatePhrase = ({
     swap(source.index, destination.index);
   };
 
+  const validatePhraseField = async (field: Record<'id', string>): Promise<boolean> => {
+    try {
+      await PhrasalTemplateResponseValuePhraseSchema.validate(field, { abortEarly: false });
```

Comment:
Curious why you thought it necessary to include `abortEarly: false`? The default is `true` because it's more performant. I ran some tests locally and it seems to work fine using the default behaviour, but maybe I missed an edge case?
```suggestion
      await PhrasalTemplateResponseValuePhraseSchema.validate(field);
```